src/ios/VLCPlugin.m

```diff
@@ -630,6 +656,24 @@ - (NSString*) vlc_convertAudioStateToString:(int) state{
 
 }
 
+- (void)vlc_resume {
+    if (self.activeMediaType == VLCPluginMediaTypeOnDemand) {
+        if (self.activePK) {
+            NSString *js = [NSString stringWithFormat:@"window.localStorage.getItem('audio-status-%@')", self.activePK];
```

Comment:
I see the value in not coupling this plugin to the NYPR app's implementation of audio status storage in the the js apps.

The established plugin provides an interface for controlling the native player from a js app, but not for the native player for requesting information from a js app. What do you think this interface should look like so that it is not again dependent on the NYPR app's implementation?

An alternative to this could be to drop the assumption that the js app stores the current position of playback at all and instead to ensure that playback resumes after a long pause in a purely native fashion. This could be done by caching the playback position on a long-pause-stop and using it to resume from the right position.